src/mongocxx/exception/exception.cpp

```diff
@@ -10,29 +10,22 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License./**
+// limitations under the License.
 
 #include <mongocxx/config/prelude.hpp>
 
-#include <cstdint>
 #include <string>
+#include <utility>
 
-// TODO: figure out if we need this
-// TODO: figure out if bson document should be view/value, etc.
+#include <mongocxx/exception/exception.hpp>
 
 namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
-namespace error {
+namespace exception {
```

Comment:
On the bsoncxx side I chose not to use a namespace exception, for two reasons: first, I wanted the type to read bsoncxx::exception, not bsoncxx::exception::exception; second, the name 'exception' being used as both a type and a namespace seemed to make the compiler unhappy.  I guess it's a matter of taste on the first part.